source/org/zfin/framework/api/TranscriptFiltering.java

```diff
@@ -0,0 +1,32 @@
+package org.zfin.framework.api;
+
+import org.zfin.marker.Transcript;
+
+public class TranscriptFiltering extends Filtering<Transcript> {
+
+
+    public TranscriptFiltering() {
+        filterFieldMap.put(FieldFilter.TYPE, typeFilter);
+        filterFieldMap.put(FieldFilter.STATUS, statusFilter);
+        filterFieldMap.put(FieldFilter.TRANSCRIPT_ID, idFilter);
+        filterFieldMap.put(FieldFilter.STATUS_EMPTY, emptyStatusFilter);
+    }
+
+    public static FilterFunction<Transcript, String> idFilter =
+        (transcript, value) -> FilterFunction.contains(transcript.getZdbID(), value);
```

Comment:
This looks good. It got me thinking that FilterFunction.contains might be better named containsAll?

Reply:
Is this because the variable 'value' can be a multi-word string and the implemented logic uses an AND operation?
Good suggestion. Will change in a future release.